CMakeLists.txt

```diff
@@ -66,7 +66,13 @@ else()
     # Note: It may conflict with the loading mechanism in onnx's CMake configuration.
     # See external/onnx/CMakeLists.txt for more details.
     include(FindProtobuf)
-    find_package(Protobuf ${PROTOBUF_VERSION} REQUIRED)
+
+    # As today, it s not sure that menoh will work perfectly with future version of protobuf (2.7.0, ...) : 
+    # https://github.com/protocolbuffers/protobuf/releases
+    # Let's be safe and require protobuf 2.6.1:
+    set(REQUIRED_PROTOBUF_VERSION "2.6.1")
+    message(STATUS "REQUIRED PROTOBUF_VERSION= ${REQUIRED_PROTOBUF_VERSION}")
```

Comment:
This `message` seems redundant because succeeding `find_package` checks the version and print similar message.